FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/Microsoft.Bot.Builder.FunctionalTests.csproj

```diff
@@ -2,7 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework Condition="'$(BuildTarget)' == 'netcoreapp31'">netcoreapp3.1</TargetFramework>
-    <TargetFrameworks Condition="'$(BuildTarget)' == ''">netcoreapp3.1</TargetFrameworks>
+    <TargetFramework Condition="'$(BuildTarget)' == 'net6'">net6.0</TargetFramework>
```

Comment:
Can we check if we need multi target here? or net 6 is enough.

Reply:
Hi @gabog, we aren't testing this project as multitarget, and we never did, so we updated it to target only net6.
We are now reviewing the rest of the feedback.